src/web/app/pages-admin/admin-search-page/admin-search-page.component.ts

```diff
@@ -26,6 +26,11 @@ export class AdminSearchPageComponent {
    */
   search(): void {
     this.accountService.searchAccounts(this.searchQuery).subscribe((resp: AdminSearchResult) => {
+      const hasStudents: boolean = !!(resp.students && resp.students.length);
+      const hasInstructors: boolean = !!(resp.instructors && resp.instructors.length);
+      if (!hasStudents && !hasInstructors) {
+        this.statusMessageService.showWarningMessage('No results found.');
+      }
       this.instructors = resp.instructors;
       for (const instructor of this.instructors) {
```

Comment:
While you're at it, can you refactor these to use `hideAllInstructorsLinks()` and `hideAllStudentsLinks()` instead

Reply:
Ok!